includes/service-providers/constant_contact/class-newspack-newsletters-constant-contact.php

```diff
@@ -292,10 +292,15 @@ private function set_access_token( $access_token = '', $refresh_token = '' ) {
 	 * Get campaign name.
 	 *
 	 * @param WP_Post $post Post object.
-	 * @return String Campaign name.
+	 *
+	 * @return string Campaign name.
 	 */
 	private function get_campaign_name( $post ) {
```

Comment:
Nit: I know it was this way before, but since these methods are the same for all service providers, can we move the `get_campaign_method` to the parent `Newspack_Newsletters_Service_Provider` class?

Reply:
Good call! Done in 25c41b166d7a77ddb87135127a2e6db8301cbadc